public/pages/workflow_detail/workflow_inputs/processors_list.tsx

```diff
@@ -94,6 +94,11 @@ export function ProcessorsList(props: ProcessorsListProps) {
     [processorId: string]: boolean;
   }>({});
 
+  // check if an index has already been indexed in the ingest pipeline
+  const hasIngestedIndex =
+    props.uiConfig?.ingest?.index?.name?.value !== undefined &&
+    props.uiConfig?.ingest?.index?.name?.value !== '';
```

Comment:
Is there another way to check this instead of relying on the prop's value? It assumes that the presence of this value means the index has been ingested, which might not always be true. For example, if the upstream component has issues, it could affect the prop value being passed down.